static/css/partials/_buttons.scss

```diff
@@ -138,15 +138,4 @@
       background-color: $light-active;
     }
   }
-
-  &.flag {
-    background-image: url("../img/icon-flag.svg");
-    &:hover {
-      background-color: $light-hover;
-    }
-
-    &:active {
-      background-color: $light-active;
-    }
-  }
```

Comment:
Leftover styles from the report shot flag that was moved to the footer in #4687.  The background image file is also removed in this PR.